flair/models/pairwise_classification_model.py

```diff
@@ -150,5 +150,4 @@ def _init_model_with_state_dict(cls, state, **kwargs):
             else state["multi_label_threshold"],
             loss_weights=state["weight_dict"],
             embed_separately=state["embed_separately"],
-            **kargs,
```

Comment:
If a class inherits from TextPairClassifier, it might need to pass other kwargs ("kargs" is of course wrong).

Reply:
good point!